### ISSUE TITLE
Inconsistent output of values for dropdown fields

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#735]
**Description**

It looks like rendering the value of a dropdown field on a submission varies depending on context. For example, if I select a dropdown field for the **Submission Title Format** form setting, it will output the value of the selected option. However, when a dropdown field is mapped for an integration, it will output the label of the selected option. It would be nice if these were consistent. For our use case, we'd prefer to have it output the label since it's intended to be human-readable and the value may or may not be. However, not sure what others' use cases would be.

### Form settings
- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.7
- Craft version: Pro 3.7.30.1
- Multi-site: No

**Comment by engram-design** [comment]
So generally speaking, dropdowns (and other option-based fields) should be using their values, never their labels. I'm not _quite_ sure why I had that set to labels, but it's now values. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.8](https://github.com/verbb/formie/releases/tag/1.5.8)
